### PR TITLE
fix tablecodec

### PR DESCRIPTION
**Pull request by dantin**
Enable namespace classifier as well

### CODE REVIEW

File: server/cluster.go [hunk 1]
```diff
@@ -110,7 +109,7 @@ func (c *RaftCluster) start() error {
 		return nil
 	}
 	c.cachedCluster = cluster
-	c.coordinator = newCoordinator(c.cachedCluster, c.s.scheduleOpt, c.s.hbStreams, namespace.DefaultClassifier)
+	c.coordinator = newCoordinator(c.cachedCluster, c.s.scheduleOpt, c.s.hbStreams, newTableNamespaceClassifier(c.cachedCluster.namespacesInfo, core.DefaultTableIDDecoder))
```
Comment (disksing) [inline server/cluster.go]: How about add a config to choose namespace classifier?

File: server/core/tablecodec.go [hunk 2]
```diff
@@ -74,3 +92,79 @@ func decodeCmpUintToInt(u uint64) int64 {
 func IsPureTableID(b []byte) bool {
 	return len(b) == len(tablePrefix)+8
 }
+
+func decodeBytes(b []byte, reverse bool) ([]byte, []byte, error) {
```
Comment (disksing) [inline server/core/tablecodec.go]: Seems we don't need to decode reversed bytes in pd.

Comment (dantin) [reply server/core/tablecodec.go]: Agree, I'll remove the unused function

File: server/cluster.go (hunk 1 above)
Comment (dantin) [reply server/cluster.go]: Agree, I'll add an option in configuration.

**Comment by disksing** [comment]
LGTM.

**Pull request merged**